docs/de/docs/index.md

```diff
@@ -189,15 +184,14 @@ async def read_item(item_id: int, q: Optional[str] = None):
     return {"item_id": item_id, "q": q}
 ```
 
-**Note**:
-
-If you don't know, check the _"In a hurry?"_ section about <a href="https://fastapi.tiangolo.com/async/#in-a-hurry" target="_blank">`async` and `await` in the docs</a>.
+**Anmerkung**:
 
+Wenn Sie es nicht wissen, lesen Sie den Abschnitt _"In a hurry?"_ über <a href="https://fastapi.tiangolo.com/async/#in-a-hurry" target="_blank">`async` und `await` in der Dokumentation</a>.
```

Comment:
Or:
`Falls Sie es nicht kennen, lesen Sie den Abschnitt _"In a hurry?"_ über`

Reply:
Ich habe in meinem Edit geschrieben: "Wenn Sie das nicht verstehen, schauen Sie sich den Abschnitt "In Eile?" über [async und await in der Dokumentation](https://fastapi.tiangolo.com/async/#in-a-hurry) an."